packages/core/test/VDocument.test.ts

```diff
@@ -2,12 +2,17 @@ import JWEditor from '../src/JWEditor';
 import { testEditor } from '../../utils/src/testUtils';
```

Comment:
Add:

- `<div><p>a[]bc</p></div>`
- `<div><p>abc</p><p>d[]ef</p></div>`
- `<div><p>a[bc</p></div><p>d]ef</p>`
- `<ul><li>abc</li><li>def<ul><li>ghi</li></ul></li><li>jkl</li></ul>` and bigger nests

And check from we3's `TestList.js`:
- ~~`ulTests`~~ (done)
- `olTests`
- `checklistTests` (for when we implement checklists)
- `conversionTests`
- `indentTests`
- `outdentTests`